tests/main_test.ts

```diff
@@ -191,6 +191,15 @@ Deno.test("/foo/:path*", async () => {
   assert(body.includes("bar/baz"));
 });
 
+Deno.test("decodes parameters", async () => {
+  const resp = await router(new Request("https://fresh.deno.dev/this has spaces"));
+  assert(resp);
```

Comment:
Just curious but why is it necessary to assert the `resp`?

Reply:
I repeated what other tests have, but I guess for this test it didn't make sense. removed ✅